modules/ROOT/pages/deployment-strategies.adoc

```diff
@@ -1,339 +1,425 @@
 
 
-== Scheduling
 
-CloudHub lets you define xref:managing-schedules.adoc[Schedules] through the Runtime Manager UI that runs your flows automatically.
 
-For apps that you deploy to servers on-premises, through any modality, this is not an option. You can achieve the same by including the xref:mule-runtime::scheduler-concept[Scheduler] element in the flows of your application.
 
-== JDK Versions
+== Anypoint Runtime Fabric
 
-The version of JDK that CloudHub implements for all apps built with Mule runtime 3.5.1 or greater is JDK 1.7. Mule runtime 3.7.0 also supports JDK 1.8. Apps built with runtime 3.5.0 or older are deployed with JDK 1.6.
 
-For apps deployed on-premises, see the xref:release-notes::mule-runtime/mule-esb.adoc[runtime release notes] of the specific runtimes you're using to know the minimum JDK supported version.
+image:logo-cloud-disabled.png[link="/runtime-manager/deploying-to-cloudhub", title="CloudHub"]
+image:logo-hybrid-disabled.png[link="/runtime-manager/deploying-to-your-own-servers", title="Hybrid"]
+image:logo-server-disabled.png[link="/runtime-manager/deploying-to-your-own-servers", title="Anypoint Platform Private Cloud Edition"]
+image:logo-rtf-active.png[link="/runtime-fabric/deploy-to-runtime-fabric", title="Anypoint Runtime Fabric"]
 
-== Automatic Security Updates
+Anypoint Runtime Fabric is a container service that automates the deployment and orchestration of Mule applications and API gateways. Runtime Fabric runs within a customer-managed infrastructure on AWS, Azure, virtual machines (VMs), and bare-metal servers.
 
-Certain updates are automatically applied to applications deployed to CloudHub. Once deployed and running, if any security patches, OS updates, or critical bug fixes are released for the selected runtime version, then you will be prompted about this change. You will be able to control exactly when each update is applied. If you take no action, updates will be applied automatically for you after 30 days to ensure your applications run with the latest security patches.
 
+Runtime Fabric contains all of the components it requires. These components, including Docker and Kubernetes, are optimized to work efficiently with Mule runtimes and other MuleSoft services.
 
-For applications that are deployed elsewhere, you must carry out these Runtime updates manually.
+To use the Runtime Fabric option, you first create a Runtime Fabric using Runtime Manager.
+Then, you install Runtime Fabric on your infrastructure.
+Finally, you deploy your applications from your on-premises Runtime Manager to either a server, server group, or cluster.
 
+The following features and limitations apply when using Runtime Manager to manage Runtime Fabric deployments:
 
-== Other Components
 
-There are some components for which CloudHub has limited support for currently:
+* *High availability*
++
+Runtime Fabric provides automatic application fail-over.
```

Comment:
Runtime Fabric also provides high availability when the user deploys 2 or more replicas per application (note: this is the same concept as workers in CloudHub)